report/renderer/report-renderer.js

```diff
@@ -85,44 +85,57 @@ export class ReportRenderer {
-    env.id = 'runtime-settings';
-    this._dom.find('.lh-env__title', footer).textContent = Util.i18n.strings.runtimeSettingsTitle;
-
-    const envValues = Util.getEnvironmentDisplayValues(report.configSettings || {});
-    const runtimeValues = [
-      {name: Util.i18n.strings.runtimeSettingsUrl, description: report.finalUrl},
-      {name: Util.i18n.strings.runtimeSettingsFetchTime,
-        description: Util.i18n.formatDateTime(report.fetchTime)},
-      ...envValues,
-      {name: Util.i18n.strings.runtimeSettingsChannel, description: report.configSettings.channel},
-      {name: Util.i18n.strings.runtimeSettingsUA, description: report.userAgent},
-      {name: Util.i18n.strings.runtimeSettingsUANetwork, description: report.environment &&
-        report.environment.networkUserAgent},
-      {name: Util.i18n.strings.runtimeSettingsBenchmark, description: report.environment &&
-        report.environment.benchmarkIndex.toFixed(0)},
-    ];
-    if (report.environment.credits && report.environment.credits['axe-core']) {
-      runtimeValues.push({
-        name: Util.i18n.strings.runtimeSettingsAxeVersion,
-        description: report.environment.credits['axe-core'],
-      });
-    }
-
-    for (const runtime of runtimeValues) {
-      if (!runtime.description) continue;
-
-      const item = this._dom.createComponent('envItem');
-      this._dom.find('.lh-env__name', item).textContent = runtime.name;
-      this._dom.find('.lh-env__description', item).textContent = runtime.description;
-      env.appendChild(item);
-    }
+    this._renderMetaBlock(report, footer);
 
     this._dom.find('.lh-footer__version_issue', footer).textContent = Util.i18n.strings.footerIssue;
     this._dom.find('.lh-footer__version', footer).textContent = report.lighthouseVersion;
     return footer;
   }
 
+  /**
+   * @param {LH.ReportResult} report
+   * @param {DocumentFragment} footer
+   */
+  _renderMetaBlock(report, footer) {
+    const envValues = Util.getEmulationDescriptions(report.configSettings || {});
+
+    const axeVersion = report.environment.credits['axe-core'];
+    const match = report.userAgent.match(/(\w*Chrome\/[\d.]+)/); // \w* to include 'HeadlessChrome'
+    const chromeVer = Array.isArray(match) ? match[1].replace('/', ' ') : 'Chrome';
+    const pageloadDurationMs = Math.max(
+      report.timing.entries.find(e => e.name === 'lh:gather:loadPage-defaultPass')?.duration || 0,
+      report.audits.interactive.numericValue || 0
+    );
+
+    // [CSS icon class, textContent, tooltipText]
+    const metaItems = [
+      ['date', `Captured at ${Util.i18n.formatDateTime(report.fetchTime)}`],
+      [
+        'devices',
+        `Lighthouse ${report.lighthouseVersion}, ${envValues.deviceEmulation}`,
+        `Host CPU Power: ${report?.environment.benchmarkIndex.toFixed(0)}. ` +
+            `CPU throttling: ${envValues.cpuThrottling}. ` +
+            `${Util.i18n.strings.runtimeSettingsAxeVersion}: ${axeVersion}`,
```

Comment:
WDYT about splitting this into three icons:

- Device emulation: Would use this "devices" icon
- CPU throttling: CPU icon (see image)
- Lighthouse version + axe version: Could use a grayscale lighthouse logo

I want to make sure our icon usage is consistent across reports. In the flow report we are currently using the "devices" icon exclusively for emulation settings:

<img width="219" alt="Screen Shot 2021-09-27 at 3 51 41 PM" src="https://user-images.githubusercontent.com/6752989/134975595-5ae13a30-9c50-40d8-b568-4da2096236b5.png">